src/Library/demos/Status Page/main.blp

```diff
@@ -0,0 +1,58 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage content {
+
+  title: "No Results Found";
+  description: _("Try a different Search.");
+  icon-name: 'edit-find-symbolic';
+
+ FlowBox {
+      homogeneous: true;
+      row-spacing: 18;
+      selection-mode: none;
+
+
+      Box {
+        orientation: vertical;
+
+
+        Label {
+          label: "Enter a search Keyword Here";
+          margin-bottom: 12;
+        }
+
+        Entry entry_1{
+          text: '';
```

Comment:
You can use the `placeholder-text` property here, to display the three search terms that will interact with the demo.